src/v/pandaproxy/probe.cc

```diff
@@ -13,25 +13,46 @@
 #include "prometheus/prometheus_sanitize.h"
 
 #include <seastar/core/metrics.hh>
+#include <seastar/core/metrics_registration.hh>
```

Comment:
I think the commit message on this needs updating.

Reply:
Adding in the metrics that are created in this commit would be great.